adicionando 2764 2765 2766 (#100)
petbccufscar/URI: Iniciante/2764/README.md

# Problema:   
O seu professor gostaria de fazer um programa com as seguintes características:

1. Leia uma data no formato DD/MM/AA;
2. Imprima a data no formato MM/DD/AA;
3. Imprima a data no formato AA/MM/DD;
4. Imprima a data no formato DD-MM-AA.

**Problema Completo**: https://www.urionlinejudge.com.br/judge/pt/problems/view/2764

# Resolução:

> Iniciaremos a resolução deste problema declarando as variaveis que iremos utilizar, serão 3 variaveis do tipo inteiro onde iremos armazenar os valores correspondentes as datas que iremos receber como entrada

Reviewer: ```suggestion
Iniciaremos a resolução deste problema declarando as 3 variáveis do tipo inteiro que iremos utilizar, nas quais serão armazenados os valores correspondentes às datas recebidas como entrada
```

```c
  int dia, mes, ano;
```

> Durante a nossa leitura através da função `scanf` ao deparar com um caractere "/" ele irá interromper a leitura e ira procurar por novos valores, salvando estes em cada variavel, dividindo em 3 blocos de numeros inteiros

Reviewer: ```suggestion
Durante a leitura (através da função `scanf`), ao deparar-se com um caractere "/", este será ignorado e os demais valores serão salvos em cada variável, dividindo-os em 3 blocos de números inteiros.
```

```c
	scanf("%d/%d/%d", &dia, &mes, &ano);
```

> Após realizada a leitura dos 3 valores inteiros iremos apenas imprimir estes seguindo a formatação proposta pelo problema

Reviewer: ```suggestion
Após realizada a leitura dos 3 valores inteiros, iremos apenas imprimi-los seguindo a formatação proposta pelo problema.
```

Reviewer: Acho que aqui é legal explicar que o argumento com esse `02` significa que o número vai ser impresso necessariamente com 2 dígitos, acrescentando zero à esquerda quando for preciso

Author: ```suggestion
Após realizada a leitura dos 3 valores inteiros, iremos apenas imprimi-los seguindo a formatação proposta pelo problema, para isso iremos utilizar `%02d`, com isso garantimos que o valor que será apresentado irá, necessariamente, possuir 2 dígitos, colocando um "0" a esquerda quando necessário
```

Reviewer: ```suggestion
Após realizada a leitura dos 3 valores inteiros, iremos apenas imprimi-los seguindo a formatação proposta pelo problema. Para isto, utilizamos o argumento `%02d`, garantindo que o valor que será apresentado irá, necessariamente, possuir 2 dígitos, colocando um "0" a esquerda quando necessário.
```
